distributed/shuffle/tests/test_merge.py

```diff
@@ -8,7 +8,7 @@
 dd = pytest.importorskip("dask.dataframe")
 import pandas as pd
 
-from dask.dataframe._compat import tm
+from dask.dataframe._compat import PANDAS_GT_200, tm
```

Comment:
nit (and out of scope): We should fix constant naming here, this should be `PANDAS_GTE_200`.